apps/gitness/src/pages/project-settings/components/FormDialogProjectDelete.tsx

```diff
@@ -0,0 +1,117 @@
+import { useState } from 'react'
+import {
+  Button,
+  Input,
+  Icon,
+  AlertDialog,
+  AlertDialogTrigger,
+  AlertDialogContent,
+  AlertDialogHeader,
+  AlertDialogTitle,
+  AlertDialogDescription,
+  AlertDialogFooter,
+  AlertDialogCancel
+} from '@harnessio/canary'
+import { FormFieldSet, MessageTheme } from '@harnessio/playground'
+import { useForm } from 'react-hook-form'
+import { z } from 'zod'
+import { zodResolver } from '@hookform/resolvers/zod'
+
+const projectDeleteSchema = z.object({
+  verification: z.string().min(1, { message: 'Please type the DELETE to verify' })
```

Comment:
NIT: message seems grammatically wrong

"To confirm, type "DELETE" in the box below"